package.json

```diff
@@ -1,28 +1,39 @@
 {
   "private": true,
   "scripts": {
+    "build:app-shared": "yarn workspace @openvaa/app-shared build && true",
+    "build:shared": "yarn build:app-shared && yarn workspace @openvaa/core build && yarn workspace @openvaa/data build && yarn workspace @openvaa/filters build && yarn workspace @openvaa/matching build && true",
     "dev": "docker compose -f docker-compose.dev.yml up -d --force-recreate --build --wait",
     "dev:attach": "docker compose -f docker-compose.dev.yml up --force-recreate --build",
     "dev:down": "docker compose -f docker-compose.dev.yml down -v --rmi all --remove-orphans",
     "dev:stop": "docker compose -f docker-compose.dev.yml stop",
     "prepare": "husky",
     "prod": "docker compose -f docker-compose.yml -p vaa-prod up -d --build",
-    "test": "playwright test"
+    "test:e2e": "yarn workspace @openvaa/tests test",
+    "test:unit": "vitest run",
+    "test:unit:watch": "vitest",
+    "format": "yarn build:app-shared && prettier --write .",
+    "format:check": "yarn build:app-shared && prettier --check .",
+    "lint:check": "yarn build:app-shared && prettier --write . && eslint --flag unstable_config_lookup_from_file --fix backend/vaa-strapi/src backend/vaa-strapi/tests frontend packages tests",
```

Comment:
In addition to the flag, we also need to painfully enumerate the folders within backend that we want to check because some of the strapi-generated folder contain eslint.configs which will cause an error – even if we mark these folders as ignored in the eslint.config in /backend/vaa-strapi! With eslint 10 we can most likely change this to just `eslint .`.